lost/src/main/java/com/mapzen/android/lost/internal/GeofencingApiImpl.java

```diff
@@ -74,16 +82,18 @@ public void disconnect() {
   private PendingResult<Status> addGeofence(LostApiClient client, Geofence geofence,
```

Comment:
I noticed the `client` param is unused when we add a geofence. We should probably be associating geofences with the client on which they were requested the same way we are with location update requests?

This way if one client is shutdown we can remove all its associated geofences but not impact those registered by another client.

Anyway just a thought that came to mind when reading this code. We don't have to deal with it in this change set but we should probably create an issue to handle at a later time.